### ISSUE TITLE
2.8.60717.93: documentation on what files are included in a package is not complete

### ISSUE THREAD

**Issue by stevenscreek** [NuGet/Home#1177]
Repro steps:

*  Create a new class library project.
*  Create a new nuspec via *nuget spec*
*  Add two content items to the project, such as:

```XML
  <ItemGroup>
    <Content Include="README1.txt">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </Content>
    <Content Include="README2.txt">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </Content>
  </ItemGroup>
```
*  Build the package with nuget pack Project.csproj.  As expected, the package contains two content entries and one lib entry.
*  Modify the nuspec to include no files:

```XML
  <files />
```

*  Build the package again.  The content entries are gone, which is expected, but the lib entry remains.
*  Modify the nuspec to include only one of the content files:

```XML
  <files>
    <file src="README1.txt" target="content" />
  </files>
```
*  Build the package again.  This time, you receive the warning *WARNING: File 'D:\temp\VSProjects\ClassLibrary1\README1.txt' is not added because the package already contains file 'content\README1.txt'*.  And, the resulting package includes both README files and one lib entry, resulting in the `<files>` section only being useful to add additional files.

In short, when packing via csproj file:
* no matter what you put in `<files>`, you get a lib entry
* `<files />` results in no content.
* Any other `<files>` entry results in additive behavior over the content in the project file.  You can have all the content from the project or none.

(we are building via pack on the csproj as we want the dependency analysis provided by `-IncludeReferencedProjects`, which we'd have to recreate ourselves if we packed via nuspec)

Note: although this behavior took a while to work out, it's actually want we needed.  Our goal was to get rid of content entries in our packages (despite them being in our project files) without resorting to recreating the functionality of `-IncludeReferencedProjects`, and `<files />` does the trick.

**Comment by yishaigalatzer** [comment]
Hey @stevenscreek this is an great summary of the behavior. The best way to add this documentation is to submit a PR for the `docs.nuget.org` at https://github.com/NuGet/NuGetDocs.

And update the following document:
https://github.com/NuGet/NuGetDocs/blob/master/NuGet.Docs/Create/Creating-and-Publishing-a-Package.md

Served from:
http://docs.nuget.org/create/Creating-and-Publishing-a-Package

**Comment by stevenscreek** [comment]
Unfortunately, I'm not going to be able to contribute it.

**Comment by yishaigalatzer** [comment]
No worries, this is a perfect start

**Comment by csharpfritz** [comment]
Added to docs for v3.3
